crates/backup-cli/src/cli.rs

```diff
@@ -1,6 +1,10 @@
+use near_primitives::types::AccountId;
+
 #[derive(clap::Parser, Debug)]
 #[command(version = env!("CARGO_PKG_VERSION"))]
 pub struct Args {
+    /// Home directory for storing backup service secrets and configuration.
+    /// Can also be set via BACKUP_HOME_DIR environment variable.
```

Comment:
I am not sure if all these lines explaining the equivalent env var are needed. Of course it would help people unfamiliar with `clap`, but I thing doing `--help` should give the same info already